test/integration/validations.test.ts

```diff
@@ -205,8 +246,20 @@ describe("validations", () => {
 
     // note we're validating multiple properties
     expect(instance.books[0].genre.errors).to.deep.equal({
-      name: "cannot be blank",
-      base: "some error"
+      name: {
+        title: "Validation Error",
+        attribute: "name",
```

Comment:
When will this not be duplicative of line 249?

Reply:
Actually meant to make this the non-client-ified version so that it’s always the same as what came from the server.